salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/B747_8_FMC_SelectWptPage.js

```diff
@@ -13,21 +13,31 @@ class B747_8_FMC_SelectWptPage {
             [""],
             [""]
         ];
+        function calculateDistance(wpt) {
+            const planeLla = new LatLongAlt(SimVar.GetSimVarValue("PLANE LATITUDE", "degree latitude"), SimVar.GetSimVarValue("PLANE LONGITUDE", "degree longitude"));
+            return Avionics.Utils.computeGreatCircleDistance(planeLla, wpt.infos.coordinates);
```

Comment:
maybe make this a static function on this class, so that it doesn't get re-declared every time ShowPage is called

Reply:
changed